cadence/contracts/LendingPool.cdc

```diff
@@ -1037,40 +1018,40 @@ pub contract LendingPool {
-            return <-LendingPool.redeemUnderlying(userCertificateCap: userCertificateCap, numUnderlyingToRedeem: numUnderlyingToRedeem)
+        access(all) fun redeemUnderlying(userCertificate: &{LendingInterfaces.IdentityCertificate}, numUnderlyingToRedeem: UFix64): @{FungibleToken.Vault} {
+            return <-LendingPool.redeemUnderlying(userCertificate: userCertificate, numUnderlyingToRedeem: numUnderlyingToRedeem)
         }
 
-        pub fun borrow(userCertificateCap: Capability<&{LendingInterfaces.IdentityCertificate}>, borrowAmount: UFix64): @FungibleToken.Vault {
-            return <-LendingPool.borrow(userCertificateCap: userCertificateCap, borrowAmount: borrowAmount)
+        access(all) fun borrow(userCertificate: &{LendingInterfaces.IdentityCertificate}, borrowAmount: UFix64): @{FungibleToken.Vault} {
+            return <-LendingPool.borrow(userCertificate: userCertificate, borrowAmount: borrowAmount)
         }
 
-        pub fun repayBorrow(borrower: Address, repayUnderlyingVault: @FungibleToken.Vault): @FungibleToken.Vault? {
+        access(all) fun repayBorrow(borrower: Address, repayUnderlyingVault: @{FungibleToken.Vault}): @{FungibleToken.Vault}? {
             return <-LendingPool.repayBorrow(borrower: borrower, repayUnderlyingVault: <-repayUnderlyingVault)
         }
 
-        pub fun liquidate(liquidator: Address, borrower: Address, poolCollateralizedToSeize: Address, repayUnderlyingVault: @FungibleToken.Vault): @FungibleToken.Vault? {
+        access(all) fun liquidate(liquidator: Address, borrower: Address, poolCollateralizedToSeize: Address, repayUnderlyingVault: @{FungibleToken.Vault}): @{FungibleToken.Vault}? {
             return <-LendingPool.liquidate(liquidator: liquidator, borrower: borrower, poolCollateralizedToSeize: poolCollateralizedToSeize, repayUnderlyingVault: <-repayUnderlyingVault)
         }
 
-        pub fun flashloan(executorCap: Capability<&{LendingInterfaces.FlashLoanExecutor}>, requestedAmount: UFix64, params: {String: AnyStruct}) {
-            LendingPool.flashloan(executorCap: executorCap, requestedAmount: requestedAmount, params: params)
+        access(all) fun flashloan(executor: &{LendingInterfaces.FlashLoanExecutor}, requestedAmount: UFix64, params: {String: AnyStruct}) {
+            LendingPool.flashloan(executor: executor, requestedAmount: requestedAmount, params: params)
         }
     }
 
     /// PoolAdmin
     ///
-    pub resource PoolAdmin: LendingInterfaces.PoolAdminPublic {
+    access(all) resource PoolAdmin: LendingInterfaces.PoolAdminPublic {
         /// Admin function to call accrueInterest() to checkpoint latest states, and then update the interest rate model
-        pub fun setInterestRateModel(newInterestRateModelAddress: Address) {
+        access(all) fun setInterestRateModel(newInterestRateModelAddress: Address) {
```

Comment:
This might need an entitlement

Reply:
No need as replied in below that all these are admin functions, won't be exposed through public capabilities in any condition